### PR TITLE
SoundWire: updates from Bard and Shuming

### PR DESCRIPTION
**Pull request by plbossart**
simple patches extracted from PR2063 for faster merge while we debug timeouts.

### CODE REVIEW

File: sound/soc/codecs/rt711.c [hunk 1]
```diff
@@ -544,9 +544,12 @@ static int rt711_set_amp_gain_put(struct snd_kcontrol *kcontrol,
 			break;
 	}
 
+	mutex_lock(&rt711->calibrate_mutex);
```
Comment (bardliao) [inline sound/soc/codecs/rt711.c:547]: @shumingfan Does rt711_set_bias_level() needs the mutex, too?

Comment (shumingfan) [reply sound/soc/codecs/rt711.c:547]: @bardliao Right, rt711_set_bias_level() should need the mutex. Thanks.

Comment (plbossart) [reply sound/soc/codecs/rt711.c:547]: @bardliao if you want to close that PR and add a new one with more fixes that's fine with me. I am running out of time.

**Comment by bardliao** [comment]
replaced by #2625

**Pull request closed without merging**